modules/thunder_liveblog/thunder_liveblog.info.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Thunder Liveblog'
```

Comment:
hmm, this wasn't planned like that, since the liveblog lazyloads, so it must be always added to the end and it should be easily possible to set the "live" flag.